core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1739,8 +1743,8 @@ object SnappySession extends Logging {
             plan.childRDD.id, true)
       case _: LocalTableScanExec =>
         (null, Array.empty[Int], -1, false) // cache plan but no cached RDD
-      case _ if allbroadcastplans.nonEmpty =>
-        (null, Array.empty[Int], -1, false) // cache plan but no cached RDD
+      // case _ if allbroadcastplans.nonEmpty =>
+        // (null, Array.empty[Int], -1, false) // cache plan but no cached RDD
```

Comment:
Ok for this commit. But if we are not going to address tokenization of bchj for 1.0 then perhaps we could add back caching with Literal for bchj as before. This can be done by changing back ParamLiteral to the Literal inside the broadcast plan sub-trees (which is equivalent of turning off tokenization for the selected portion of query). Can you track this for GA (either add tokenization for bchj which is preferred or else if too much work then change ParamLiteral to Literal instead of skipping caching entirely)?